pkg/operators/oci-handler/oci.go

```diff
@@ -123,6 +124,12 @@ func (o *ociHandler) InstanceParams() api.Params {
 			Description: "Secret to use when pulling the gadget image",
 			TypeHint:    api.TypeString,
 		},
+		{
+			Key:         "annotate",
```

Comment:
Please document this in https://www.inspektor-gadget.io/docs/latest/gadget-devel/metrics.

Reply:
will be removed from this PR once it found a better home (#3468 ?)